src/Umbraco.Tests/Web/UmbracoHelperTests.cs

```diff
@@ -28,8 +28,10 @@ public void TearDown()
 
         private void SetUpDependencyContainer()
```

Comment:
Seems to be dead code :)

Reply:
Aye that entire class is dead infact, but I have removed Current.Reset and all calls in 384ae54